x-pack/test/cloud_security_posture_functional/agentless/create_agent.ts

```diff
@@ -128,6 +129,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       await cisIntegration.clickOptionButton(AWS_SINGLE_ACCOUNT_TEST_ID);
 
       await cisIntegration.inputIntegrationName(integrationPolicyName);
+      await pageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
I don't think checking for "Loading"  on the header component is desirable for those scenarios. Relying on the behaviour of the header component seems to make us very prone to flakiness. 

For a more flaky-proof solution, we should consider wrapping async side effects with `retry.try` instead of using `pageObjects.header.waitUntilLoadingHasFinished()`,

more reference on: https://github.com/elastic/security-team/issues/10975